packages/p2p/src/components/file-dropzone/file-dropzone.scss

```diff
@@ -7,7 +7,7 @@
     transform: translate3d(0, 0, 0);
     transition: transform 0.25s ease, opacity 0.15s linear;
 
-    @include mobile {
+    @include mobile-or-tablet-screen {
         max-width: 26rem;
```

Comment:
just be cautious with using this, cause sometimes it has to follow desktop sizing on tablet, and sometimes mobile, just cross check between v1 and v2, so when QA eventually test the two, they are "similar"

Reply:
yeah.. for the layout changes i used the mobile design for tablet.. but for the font size and icon size, i'm using the desktop ones for the tablet... but there are quite a lot changes and its difficult to track 😅

Reply:
yeah the designs should honestly just follow mobile sizes for tablet tbh, makes it a whole lot easier for us